### DISCUSSION TITLE
Restrict form to remove last item in dynamic fields

### DISCUSSION THREAD

**Discussion by abdulmateentechbits** [react-hook-form/react-hook-form#9386] [Q&A] [answered]
Hi friends!
I hope all of you fine.
I want a user can not delete the last child of defaults array field i want last item should be their 
Here is my codesandbox link 
https://codesandbox.io/s/restrict-user-to-remove-default-field-bxw8kz?file=/src/App.js
Thankyou!

**Comment by bluebill1049** [comment, marked as answer]
couldn't you check the fields length and then just disable the button?

**Comment by abdulmateentechbits** [comment]
Thanks a lot it's work
